src/Compilers/CSharp/Test/Emit3/Attributes/AttributeTests_Embedded.cs

```diff
@@ -267,15 +608,35 @@ public void M(in int p)
         [Fact]
         public void CompilerShouldIgnorePublicEmbeddedAttributesInReferencedAssemblies()
         {
-            var reference = CreateCompilation(assemblyName: "testRef", parseOptions: TestOptions.Regular.WithNoRefSafetyRulesAttribute(), source: @"
-namespace Microsoft.CodeAnalysis
-{
-    public class EmbeddedAttribute : System.Attribute { }
-}
-namespace OtherNamespace
-{
-    public class TestReference { }
-}").ToMetadataReference();
+            var reference = CompileIL("""
+                .assembly extern mscorlib { }
+                .assembly testRef
+                {
+                }
+
+                .class public auto ansi beforefieldinit OtherNamespace.TestReference
+                    extends [mscorlib]System.Object
+                {
+                    // Methods
+                    .method public hidebysig specialname rtspecialname instance void .ctor () cil managed 
+                    {
+                        ldarg.0
+                        call instance void [mscorlib]System.Object::.ctor()
+                        ret
+                    }
+                }
+
+                .class public auto ansi beforefieldinit Microsoft.CodeAnalysis.EmbeddedAttribute
```

Comment:
>EmbeddedAttribute [](http://example.com/codeflow?start=79&length=17)

Why wasn't this `EmbeddedAttribute` definition used for `Test.M()` below? (This definition doesn't appear to be marked `[Embedded]`.)